setup.py

```diff
@@ -26,7 +26,10 @@
     packages=setuptools.find_packages(),
     install_requires=install_requires,
     entry_points={
-        'console_scripts': "starfish=starfish.starfish:starfish"
+        'console_scripts': [
+            "starfish=starfish.starfish:starfish",
+            "build_sample_experiment=starfish.experiment.builder.cli:main",
```

Comment:
I think b_s_e would surprise me if it showed up on `$PATH` after `pip install starfish`

Reply:
Yep, I had similar thoughts. I think it might make more sense to include it as a sub-module of starfish. For now, tagging it with a name that associates with our package is enough for me because I want to get this out to collaborators. 

Suggestion: 
`build_sample_starfish_experiment` + new issue to include it in the starfish binary.